django/utils/encoding.py

```diff
@@ -94,7 +94,7 @@ def force_bytes(s, encoding='utf-8', strings_only=False, errors='strict'):
         if encoding == 'utf-8':
             return s
         else:
-            return s.decode('utf-8', errors).encode(encoding, errors)
+            return s.decode(errors).encode(encoding, errors)
```

Comment:
I'll revert that (unless `errors=errors` seems better) and I added a test for this path in https://github.com/django/django/pull/8037.